samples/notebooks/Lightweight Python components - basics.ipynb

```diff
@@ -236,13 +250,7 @@
     "#Get or create an experiment and submit a pipeline run\n",
     "import kfp\n",
     "client = kfp.Client()\n",
-    "list_experiments_response = client.list_experiments()\n",
```

Comment:
Why is this code being removed?

Reply:
The test needs to know the experiment name in order to check the status for test results.